hawc/apps/animal/api.py

```diff
@@ -93,10 +91,12 @@ def study_heatmap(self, request, pk):
         if unpublished and not self.assessment.user_is_reviewer_or_higher(self.request.user):
             raise PermissionDenied("You must be part of the team to view unpublished data")
         key = f"assessment-{self.assessment.id}-bioassay-study-heatmap-pub-{unpublished}"
-        df = cache.get(key)
-        if df is None:
-            df = models.Endpoint.heatmap_study_df(self.assessment, published_only=not unpublished)
-            cache.set(key, df, settings.CACHE_1_HR)
+        df = cacheable(
+            lambda: models.Endpoint.heatmap_study_df(
```

Comment:
this is super minor, but the indentation is throwing me off. Can we do something like instead?

```python
# one liner
func = lambda: models.Endpoint.heatmap_study_df(self.assessment, published_only=not unpublished)
# or this

def func():
    return models.Endpoint.heatmap_study_df(self.assessment, published_only=not unpublished)
```